src/net/sourceforge/kolmafia/session/LoginManager.java

```diff
@@ -215,8 +215,9 @@ public static void initialize(final String username) {
 
     if (Preferences.getString("spadingData").length() > 10) {
       KoLmafia.updateDisplay(
-          "Some data has been collected that may be of interest "
-              + "to others.  Please type `spade' to examine and submit or delete this data.");
+          "Some data has been collected that may be of interest to others. "
+              + "Please type `spade' to examine and submit or delete this data. If you have a lot, "
```

Comment:
Please type `spade' to examine and optionally submit the data or `spade autoconfirm` to submit all of the spaded data.  Either way the data will be deleted whether shared or not.

Not sure that is really better but it is clearer.

Reply:
updated, thanks